server/index.ts

```diff
@@ -34,41 +27,59 @@ const firebaseConfig = {
   measurementId: 'G-TCPV9WSHNH'
 };
 
-// const app = express();
-// app.use(bodyParser.urlencoded({ extended: false }));
-// app.use(bodyParser.json());
+const firebaseApp = initializeApp(firebaseConfig);
+const auth = getAuth(firebaseApp);
+const provider = new GoogleAuthProvider();
+const db = getFirestore(firebaseApp);
+const storage = getStorage(firebaseApp);
 
-// const PORT = process.env.PORT || 8080;
+// create object, tired of messing with all these import/exports
+const ourAuth = {
+  auth: auth,
+  provider: provider,
+  googleAuthProvider: GoogleAuthProvider,      // need this static reference to call some methods
+  getAdditionalUserInfo: getAdditionalUserInfo,
+  signInWithPopup: signInWithPopup,
+  signInWithRedirect: signInWithRedirect,
+  getRedirectResult: getRedirectResult,
+  getAuth: getAuth,
+  createUserWithEmailAndPassword: createUserWithEmailAndPassword,
+  signInWithEmailAndPassword: signInWithEmailAndPassword,
+  signOut: signOut
+};
 
-// app.get('/', (req: Request, res: Response) => {
-//     res.send('hello');
-// })
+const ourFirestore = {
+  db: db,
+  collection: collection,
+  doc: doc,
+  addDoc: addDoc,
+  getDoc: getDoc,
+  setDoc: setDoc
+};
 
-// app.listen(PORT, () => {
-//   console.log(`Server listening on port ${PORT}...`);
-// });
+const ourStorage = {
+  storage: storage,
+  ref: ref,
+  uploadBytes: uploadBytes
+};
 
-const firebaseApp = initializeApp(firebaseConfig);
+/*const app = express();
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(bodyParser.json());
 
-const db = getFirestore(firebaseApp);
+const PORT = process.env.PORT || 8080;
 
-// Initialize Firebase
-const auth = getAuth(firebaseApp);
-const storage = getStorage(firebaseApp);
+app.get('/', (req: Request, res: Response) => {
```

Comment:
if this is to be uncommented and the intent is to have an express server, it won't function without the below app.listen

Reply:
ok I see this is commented out